clients/python-wrapper/lakefs/object_io.py

```diff
@@ -70,10 +97,16 @@ def read(self, read_bytes: int = None) -> bytes:
         self._pos = new_pos  # Update pointer position
         return contents
 
-    def stat(self):
+    def stat(self) -> lakefs_sdk.ObjectStats:
```

Comment:
I suggest we do not return generated types.

Reply:
Do you suggest creating our own implementation for ObjectStats, Refs, 'X'List etc?